docs/semgrep-ci.md

```diff
@@ -24,10 +25,153 @@ See [how to set it up with your CI provider](sample-ci-configs.md).
+on the [Projects page](https://semgrep.dev/manage/projects) of Semgrep App
+
+!!! info
+    This page will list only the repositories
+    that Semgrep has permission to see.
+    You can add repositories on your organization's settings page on GitHub.
+    Just go to Settings > Installed GitHub Apps > semgrep.dev > Configure
+    and make your changes in the 'Repository access' section.
+  
+You will get a chance to configure a few settings,
+such as whether you want to run on pushes, on pull requests, or on both.
+We recommend using Semgrep with the pre-selected settings.
+When you're done, Semgrep will commit a CI workflow file to your repository.
+
+!!! warning
+    Semgrep cannot commit this file if there are
+    branch protection rules preventing pushes to your default branch.
+    In this case, you can temporarily disable your branch protection rules,
+    or follow our guide for manual setup.
+
+## Manual setup
+
+If you're using GitHub Actions,
+you can generate and copy a CI configuration file on Semgrep App as above,
+and then commit it manually.
+
+If you're using any other CI provider,
+you can use one of our [sample CI configuration files](sample-ci-configs.md).
+
+# Configuration
+
+Semgrep CI is configured with command line flags.
+This is a reference for most common options,
+but you can see all available settings with `--help`.
+
+!!! info
+    When you're configuring the semgrep-action,
+    you set these options under the `with:` block
+    of your workflow YAML file.
+
+    Option names are camel-cased in the workflow configuration file.
+    This means that `--audit-on push` changes to `auditOn: push`.
+
+## Selecting rules to scan with
+
+The `--config` flag lets you choose
+what rules and patterns semgrep should scan for.
+You can set specify rules in one of the following ways:
+
+- **semgrep.dev ruleset ID**: `config: p/r2c`  
+  referring to a ruleset found on [semgrep.dev's rulesets page](https://semgrep.dev/explore)
+- **semgrep.dev snippet ID**: `config: s/xYz` or `config: s/john:named-rule`
+  referring to a rule published from the [semgrep.dev playground](https://semgrep.dev/editor)
+- **semgrep.dev registry ID**: `config: r/python.flask`  
+  referring to a subset of the [semgrep.dev rule registry](https://semgrep.dev/r)
+
+If `config` is unset,
+the default behavior is to look for rules
+in the `.semgrep.yml` file in your repo,
+or load all rules from the `.semgrep/` directory in your repo.
+
+If none of these provide a configuration,
+Semgrep CI will exit with a failing status code.
 
-## Technical Details
+## Connecting to Semgrep App
 
-### Packaging
+To use your Semgrep App account,
+set `--publish-deployment` and `--publish-token`.
+These act as your username and password for authentication.
+You can find the right values for these variables
+on the [Dashboard > Settings](https://semgrep.dev/manage/settings) page.
+
+## Ignoring files & directories
+
+You can commit a `.semgrepignore` file
```

Comment:
Must this file be in the root of the project or will Semgrep CI find it in a subfolder?